workspaces/Data_unconstrainedFpt_withJets/step0

```diff
@@ -280,6 +316,8 @@ for jet in jet_variations:
                     fake_systematics.add(emu_mue=em_me, sysname=sys, jetvariation=jet, l1pt=l1pt_key, updown=(norm_up, norm_do))
                     h_do.Scale(1.0/norm_do)
                     h_up.Scale(1.0/norm_up)
+                    h_do.Write() #overwrite with normalized histo?
```

Comment:
Do we want to save the syst histograms before or after the normalization change?
I believe after, but in the original code this is not clear to me
https://github.com/gerbaudo/hlfv-fitmodel/blob/35d1e51a0ebc3b5b7ffefad027d5b3612ce2c16d/workspaces/Data_unconstrainedFpt_withJets/step0#L782